fizz-buzz.test.js

```diff
@@ -1,6 +1,40 @@
-const { fizzBuzz } = require('./fizz-buzz');
+const { fizzBuzz } = require("./fizz-buzz");
 
 // 1. write a describe to contain our tests
 // 2. add a first test. You could use one of the console.log tests in the other file as a starting point
 // 3. run the test with `npm test` to see if it passes
 // 4. add another test, and another, and another, until you've tested all the functionality of the fizzBuzz function
+
+// test("should if the number is divisible by 5, return Buzz", () => {
+//   const result = fizzBuzz(15);
+//   expect(result).toEqual("FizzBuzz");
+// });
+
+describe("FizzBuzz", () => {
```

Comment:
Usually, you'd group several tests together in a single describe() (ie this one, and remove the others in this file). The label for the describe could be the name of the function eg fizzBuzz.
So, one `describe` wrapping the four `it` tests.